ci/update-requirements.py

```diff
@@ -76,6 +76,10 @@ def pc(target, *sources):
         'demos/python_demos/requirements.txt', openvino_dir / 'python/requirements.txt')
     pc('ci/requirements-downloader.txt',
         'tools/downloader/requirements.in')
+    pc('ci/requirements-quantization.txt',
+        'tools/accuracy_checker/setup.py',
```

Comment:
Please use `requirements-core.in`, and `requirements.in` instead (see the call that generates `requirements-ac.txt`).

Reply:
Done. (Downloader Job [logs](http://213.221.44.203/job/omz/job/precommit/job/downloader_ubuntu18/466/console))